core/web/jobs_controller_test.go

```diff
@@ -92,8 +95,72 @@ func TestJobsController_Create_ValidationFailure_OffchainReportingSpec(t *testin
 	}
 }
 
-func TestJobController_Create_HappyPath(t *testing.T) {
+func TestJobController_Create_HappyPathOCR(t *testing.T) {
```

Comment:
The OCR test was failing after adding the eth client mock, although ideally these tests wouldn't have to start the entire chainlink application.